### DISCUSSION TITLE
Automatically verify spectrograms

### DISCUSSION THREAD

**Discussion by mat926** [RogueOneEcho/caesura#95] [Ideas]
As part of the transcode process, could Caesura run [nectar](https://gitlab.com/passelecasque/nectar.git) or any similar tool to ensure the files are not bad/lossy transcodes?

**Comment by RogueOneEcho** [comment]
If it's technically feasible I'd certainly consider it, but I'm sceptical that any integration would be reliable. 

It looks like nectar is a python project that hasn't been touched in 6 years, and the README doesn't inspire much confidence that it worked even then.

**Reply by RogueOneEcho** [reply]
Closing and marking as won't fix as I don't believe it's technically possible and haven't received a response.

Feel free to suggest reasons for it to be re-opened
